foxsi.pro

```diff
@@ -0,0 +1,42 @@
+; June 2016
+; STARTUP FOXSI
+; Description: 
+;
+;
+
+PRO FOXSI, YEAR
+
+COMMON FOXSI
```

Comment:
As discussed during the telecon, I think the path forward we agreed on was to use txt files for as much as is appropriate.